.changeset/upgrade-dependencies-august.md

```
@@ -0,0 +1,14 @@
+---
+'ciderpress': patch
+'@ciderpress/cli': patch
+'@ciderpress/config': patch
+'@ciderpress/templates': patch
+'@ciderpress/theme': patch
+'@ciderpress/ui': patch
+---
+
+Upgrade runtime and build dependencies to their latest compatible releases.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)